public/app/features/explore/state/actions.ts

```diff
@@ -466,171 +452,54 @@ export function runQueries(exploreId: ExploreId): ThunkResult<void> {
     stopQueryState(queryState, 'New request issued');
 
     queryState.sendFrames = true;
-    queryState.sendLegacy = true; // temporary hack until we switch to PanelData
+    queryState.sendLegacy = true;
```

Comment:
Without the comment this is bit more confusing. Why do we need both right now?

Reply:
My bad, we're still using legacy in this PR

Reply:
I found another reason why we must use `sendLegacy` and that's because we send the received series to Angular query editors with the `eventBridge.emit(...)` and those expect legacy data formats, right @torkelo @ryantxu?

Reply:
Yes -- in dashboards, when you open the query editor it subscribes to the results with both formats, sending DataFrames to react panels and legacy to angular panels:
https://github.com/grafana/grafana/blob/master/public/app/features/dashboard/panel_editor/QueriesTab.tsx#L66

Since explore always has the editor open, getting both will make sense.  Later we could maybe check that an angular query editor exists and change the subscription based on that.